### ISSUE TITLE
[Rule] Empty lines in multiline statements

### ISSUE THREAD

**Issue by bhirsz** [MarketSquare/robotframework-robocop#371]
This should trigger warning:
```
Keyword    ${arg}
...    1

...    2
```
